cinema/models.py

```diff
@@ -5,6 +5,32 @@
     title = models.CharField(max_length=255)
     description = models.TextField()
     duration = models.IntegerField()
+    genres = models.ManyToManyField("Genre", blank=True, related_name="genres")
+    actors = models.ManyToManyField("Actor", blank=True, related_name="actors")
```

Comment:
Similar to the `genres` field, the `related_name` for the `actors` field should be unique and descriptive. Using 'actors' as the related name can cause confusion and potential conflicts. Consider using something like 'movies' to indicate the reverse relationship from `Actor` to `Movie`.